apps/api/src/app/inbox/dtos/bulk-update-preferences-request.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { IsArray, IsDefined, IsString, ValidateNested } from 'class-validator';
+import { Type } from 'class-transformer';
+
+import { UpdatePreferencesRequestDto } from './update-preferences-request.dto';
+
+export class BulkUpdatePreferenceItemDto extends UpdatePreferencesRequestDto {
+  @IsDefined()
+  @IsString()
+  readonly workflowIdOrInternalId: string;
```

Comment:
Do we have this pattern in other places? wonder if it's better to just stick to workflowId, it's hard for me to see if anyone will ever user the internal id for this